ReactNativeClient/lib/models/Setting.js

```diff
@@ -253,6 +263,57 @@ class Setting extends BaseModel {
 					return output;
 				},
 			},
+			preferredLightTheme: {
+				value: Setting.THEME_LIGHT,
+				type: Setting.TYPE_INT,
+				public: true,
+				appTypes: ['desktop'],
+				isEnum: true,
+				label: () => _('Preferred Light Theme'),
+				description: () =>
+					_('Applicable for Light System Theme and Night Mode button'),
+				section: 'appearance',
+				options: () => {
+					const output = {};
+					output[Setting.THEME_LIGHT] = _('Light');
+					output[Setting.THEME_DARK] = _('Dark');
+					if (platform !== mobilePlatform) {
+						output[Setting.THEME_DRACULA] = _('Dracula');
+						output[Setting.THEME_SOLARIZED_LIGHT] = _('Solarised Light');
+						output[Setting.THEME_SOLARIZED_DARK] = _('Solarised Dark');
+						output[Setting.THEME_NORD] = _('Nord');
+					} else {
+						output[Setting.THEME_OLED_DARK] = _('OLED Dark');
+					}
+					return output;
+				},
+			},
+			preferredDarkTheme: {
+				value: Setting.THEME_DARK,
+				type: Setting.TYPE_INT,
+				public: true,
+				appTypes: ['desktop'],
+				isEnum: true,
+				label: () => _('Preferred Dark Theme'),
+				description: () =>
+					_('Applicable for Dark System Theme and Night Mode button'),
+				section: 'appearance',
+				options: () => {
+					const output = {};
+					output[Setting.THEME_LIGHT] = _('Light');
+					output[Setting.THEME_DARK] = _('Dark');
+					if (platform !== mobilePlatform) {
+						output[Setting.THEME_DRACULA] = _('Dracula');
+						output[Setting.THEME_SOLARIZED_LIGHT] = _('Solarised Light');
+						output[Setting.THEME_SOLARIZED_DARK] = _('Solarised Dark');
+						output[Setting.THEME_NORD] = _('Nord');
+					} else {
+						output[Setting.THEME_OLED_DARK] = _('OLED Dark');
+					}
+					return output;
```

Comment:
Could you not duplicate this code three times please?

Reply:
Ohh I'm sorry I will try to reduce this code